src/commands/slash/InspectCommand.ts

```diff
@@ -1,28 +1,23 @@
-import {ColorResolvable, GuildMember, Message, MessageEmbed} from "discord.js";
-import DiscordUtils from "../../utils/DiscordUtils";
-import Command from "../../abstracts/Command";
-import DateUtils from "../../utils/DateUtils";
+import {Discord, Slash, SlashOption} from "discordx";
+import {MessageEmbed, ColorResolvable, CommandInteraction, GuildMember} from "discord.js";
 import getConfigValue from "../../utils/getConfigValue";
 import GenericObject from "../../interfaces/GenericObject";
+import DateUtils from "../../utils/DateUtils";
+import DiscordUtils from "../../utils/DiscordUtils";
 
-class InspectCommand extends Command {
-	constructor() {
-		super(
-			"inspect",
-			"Show information about a given user"
-		);
-	}
-
-	async run(message: Message, args: string[]) {
-		const userObj = args.length > 0
-			? await DiscordUtils.getGuildMember(args[0], message.guild!)
-			: message.member!;
+@Discord()
+class InspectCommand {
+	@Slash("inspect")
+	async onInteract(
+		@SlashOption("user", {type: "MENTIONABLE", required: false}) userID: GuildMember,
+			interaction: CommandInteraction): Promise<void> {
```

Comment:
I think this would look neater if it was formatted as:
```suggestion
	async onInteract(
		@SlashOption("user", {type: "USER", required: false}) userID: GuildMember,
		interaction: CommandInteraction
	): Promise<void> {
```

Also, changing `MENTIONABLE` to `USER` so that it doesn't autocomplete roles too.